src/dotnet/commands/dotnet-build/CompilerIOManager.cs

```diff
@@ -109,12 +109,7 @@ private static void AddLockFile(ProjectContext project, List<string> inputs)
                 throw new InvalidOperationException(errorMessage);
             }
 
-            inputs.Add(project.LockFile.LockFilePath);
-
-            if (project.LockFile.ExportFile != null)
```

Comment:
> Which we don't need anymore.

We need it for .xproj => .csproj references right? Aren't we still supporting project.json/.xproj in the current code?  If not, we should remove the project-model-server again.